ui/components/component-library/picker-network/picker-network.tsx

```diff
@@ -27,6 +28,7 @@ export const PickerNetwork: PickerNetworkComponent = React.forwardRef(
   <C extends React.ElementType = 'button'>(
     {
       className = '',
+      avatarGroupProps,
```

Comment:
Could we add a unit test for this prop to ensure the testing coverage is 100% 🙏

Reply:
Done!